src/Core/ProtocolDefines.h

```diff
@@ -45,6 +45,10 @@
 
 #define DBMS_MIN_REVISION_WITH_CUSTOM_SERIALIZATION 54454
 
+#define DBMS_MIN_REVISION_WITH_HTTP_HOST_IN_CLIENT_INFO 54450
```

Comment:
I have concerns about that numbers. I see that VERSION_REVISION is defined as 54465 now.
I'm verifying that part right now. I'll return soon.